package.json

```diff
@@ -2,8 +2,12 @@
   "name": "tube-tracker-back-end",
   "version": "1.0.0",
   "description": "",
-  "main": "app.js",
+  "main": "./dist/app.js",
   "scripts": {
+    "build": "tsc",
+    "dev": "ts-node ./lib/app.ts",
+    "start": "nodemon ./dist/app.js",
```

Comment:
You can actually combine `dev` and `start`:

https://stackoverflow.com/questions/37979489/how-to-watch-and-reload-ts-node-when-typescript-file-changes

That way, we can run TypeScript directly without needing to compile to JS first.